hack/verify-test-code.sh

```diff
@@ -83,6 +92,20 @@ if [ ${#errors_expect_error[@]} -ne 0 ]; then
   exit 1
 fi
 
+if [ ${#errors_expect_no_equal[@]} -ne 0 ]; then
+  {
+    echo "Errors:"
+    for err in "${errors_expect_no_equal[@]}"; do
+      echo "$err"
+    done
+    echo
+    echo 'The above files need to use framework.ExpectNotEqual(foo, bar) instead of '
```

Comment:
We have already check for `framework.ExpectError()` at line 79.
So I am fine to add this check of `framework.ExpectNotError()` as the same manner.